src/doc.cpp

```diff
@@ -1538,16 +1538,19 @@ int Doc::PrepareLyricsEnd(FunctorParams *functorParams)
 {
     PrepareLyricsParams *params = dynamic_cast<PrepareLyricsParams *>(functorParams);
     assert(params);
-
-    if ((params->m_currentSyl && params->m_lastNote) && (params->m_currentSyl->GetStart() != params->m_lastNote)) {
-        params->m_currentSyl->SetEnd(params->m_lastNote);
+    auto params_m_currentSyl = params->m_currentSyl;
```

Comment:
This line is useless, isn't it?

Reply:
I meant: you do not need another local variable

Reply:
No, we don't need extra variable.
But. 
1) Pointer dereference isn't free. If it's used twice it's not guaranteed compiler will optimise this. But it's not the point.
2) Point is it simplifies code reading and understanding.

Reply:
I find it more confusing. Having a check and early return is OK but without the extra variable.